File: Daily Test/Java/Increasing Word Matrix Pattern 24.10.2025.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        String s = "abcdefghijklmnopqrstuvwxyz";
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        for (int i = 1; i <= n; i++) {
            for (int j = i; j < n + i; j++) {
                if (j < 26) {
                    System.out.print(s.substring(0, j));
                } else {
                    int len = j % 26;
                    System.out.print(s.substring(0, len) + s.substring(0, len));
                }
                System.out.print(" ");
            }
            System.out.println();
        }
    }
}

File: Daily Test/Python/Increasing Word Matrix Pattern 24.10.2025.py
n = int(input())
s = 'abcdefghijklmnopqrstuvwxyz'
for i in range(1, n + 1):
    for j in range(i, n + i):
        if j < 26:
            print(s[:j], end=' ')
        else:
            print(s[:j % 26] + s[:j % 26], end=' ')
    print()